Spigot-API-Patches/0283-Add-EntityExtinguishEvent.patch

```diff
@@ -0,0 +1,90 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Aurora <aurora@relanet.eu>
+Date: Sat, 20 Mar 2021 18:14:24 +0100
+Subject: [PATCH] Add EntityExtinguishEvent
+
+
+diff --git a/src/main/java/io/papermc/paper/event/entity/EntityExtinguishEvent.java b/src/main/java/io/papermc/paper/event/entity/EntityExtinguishEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..11097414867a9aaeccad262b6186d2cf8ba00c29
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/event/entity/EntityExtinguishEvent.java
+@@ -0,0 +1,78 @@
++package io.papermc.paper.event.entity;
++
++import org.bukkit.entity.Entity;
++import org.bukkit.event.Cancellable;
++import org.bukkit.event.HandlerList;
++import org.bukkit.event.entity.EntityEvent;
++import org.jetbrains.annotations.NotNull;
++
++/**
++ * Is called when a burning {@link org.bukkit.entity.Entity} is extinguished.
```

Comment:
Nitpick: I would specify what "burning" and "being extinguished" means. Meaning that I would add JavaDoc links to the fireTicks property.